src/test/java/com/shima/todo_list/mapper/TodoListMapperTest.java

```diff
@@ -50,6 +51,18 @@ class TodoListMapperTest {
         Optional<TodoList> todoLists = todoListMapper.findById(1);
         assertThat(todoLists).contains(new TodoList(1, "構想", LocalDate.of(2023, 12, 6), null, null));
     }
+
+    @Test
+    @Sql(
+            scripts = {"classpath:/sqlannotation/delete-todolists.sql", "classpath:/sqlannotation/insert-todolists.sql"},
+            executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD
+    )
```

Comment:
ここもDataSetで良いと思うのですが、いかがでしょうか？
```suggestion
    @DataSet(value = "datasets/todolists.yml")
```

Reply:
`DataSet` に修正し対応致しました。

00367192352117bca64e126c77bab729c22521e1


### 動作確認


![219C20F2-AE7A-4CB0-8C2D-3D8E811D596C](https://github.com/simamumu89/Todo_API/assets/142807674/56720b1c-601c-439f-8796-99443bddb20e)